packages/documentation/package.json

```diff
@@ -22,11 +22,11 @@
     "@docusaurus/preset-classic": "2.4.0",
     "@docusaurus/theme-live-codeblock": "2.4.0",
     "@mdx-js/react": "^1.6.22",
-    "@siemens/html-test-app": "~2.0.4-beta.0",
-    "@siemens/ix": "~2.0.4-beta.0",
-    "@siemens/ix-echarts": "~2.0.4-beta.0",
+    "@siemens/html-test-app": "~2.0.4",
+    "@siemens/ix": "~2.0.4",
+    "@siemens/ix-echarts": "~2.0.4",
     "@siemens/ix-icons": "2.0.0-beta.0",
```

Comment:
Ups 👀 this should be `@siemens/ix-icons@^2.0.1`.
We should fix this for the documentation deployment. Release not affected.